content/testing.md

```diff
@@ -1,23 +1,158 @@
 # Testing Guide
 
-TODO: Explain why it is essential to have proper testing in our projects and for PRs to maintain tests up-to-date.
+At Optakt, we consistently write tests to ensure a reliable engineering environment where quality is paramount. Over the course of the product development life cycle, testing saves time and money, and helps developers write better code, more efficiently. Untested code is fragile, difficult to maintain and becomes questionable as soon as changes are made.
```

Comment:
General question - does documentation generation work with a 'single sentence per line' style?

Reply:
Yes it does. I forgot about using that here, but it's definitely something I should add to the docs style guide and do throughout these docs! Nice catch :)